app/assets/stylesheets/master.css.scss

```diff
@@ -0,0 +1,21 @@
+.hero-image {
+  background: url('/assets/background-img.jpeg') no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.enter {
+  font-family: "Special Elite";
```

Comment:
Properties should be ordered color, font-family, font-size, height, left, margin, padding, position, top, width